Bug fix on statsd port assignment

juanbrein opened the pull request:
host variable was being assigned before port. So port was always undefined and therefore falling back to 8125

juliangruber commented:
this works as expected:

```js
var host = 'foo:bar'
var port = host.split(':')[1]
var host = host.split(':')[0]
console.log(host, port)
// => foo bar
```

after your patch, in line 12, `host` won't have the `:` any more.

What parameters are you giving `statistik` exactly? The error is probably somewhere else.

juanbrein commented:
Don't understand why tests are failing but if you tell me how to fix it I'll take a look

juanbrein commented:
sorry, my mistake, however when I install statsc I can see the bug:

```

root@afea7e946413:~# npm install -g statsc
/usr/bin/statsc-server -> /usr/lib/node_modules/statsc/bin/statsc-server
statsc@0.2.5 /usr/lib/node_modules/statsc
└── statistik@0.0.7 (commander@1.0.5)
root@afea7e946413:~# 

root@afea7e946413:~# head -15 /usr/lib/node_modules/statsc/node_modules/statistik/index.js
var dgram = require('dgram');

/**
 * Returns an instance of statistik bound to `host`.
 * 
 * @constructor
 * @param {string} [host='localhost'] StatsD's hostname 
 */
var Statistik = function(host, port) {
  if (host && host.search(':')>-1) {
    host = host.split(':')[0];
    port = host.split(':')[1];
  }
  this.host = host || 'localhost';
  this.port = port || 8125;

```

trying to fix it I inverted your code which was actually right. You actually fixed it here: 0378e761cd226aa4dcca73a94a02630fc841263d

I guess statsc will have to bump statistik version

Thanks and sorry for the mess

Juan
